### PR TITLE
LPS-124480 Remove PluginContext provider as these components are not using AppContext

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by sandrodw3** [comment]
cc @p2kmgcl

File: modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/sidebar/MultiPanelSidebar.es.js [hunk 1]
```diff
@@ -43,69 +41,45 @@ export default function MultiPanelSidebar({
 	sidebarPanels,
 	variant = 'dark',
 }) {
-	const [{sidebarOpen, sidebarPanelId}, dispatch] = useContext(AppContext);
+	const [, dispatch] = useContext(AppContext);
+	const [sidebarPanelId, setSidebarPanelId] = useState('fields');
```
Comment (p2kmgcl) [inline modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/sidebar/MultiPanelSidebar.es.js:45]: If you are always updating these two `useState` all together, you can merge those states into `sidebarState` or something like that:

```js
const [{sidebarPanelId, sidebarOpen}, setSidebarState] = useState(
  {sidebarPanelId: 'fields', sidebarOpen: true}
);

...

setSidebarState({ sidebarPanelId: '???', sidebarOpen: false });
```

File: modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/plugins/rules-sidebar/index.es.js [hunk 2]
```diff
@@ -14,25 +14,17 @@
 
 import React from 'react';
 
-import {Component} from '../PluginContext.es';
```
Comment (p2kmgcl) [inline modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/plugins/rules-sidebar/index.es.js:17]: I think you can delete PluginContext.es.js file too

**Review by p2kmgcl — CHANGES_REQUESTED** [review]

File: modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/plugins/rules-sidebar/index.es.js (hunk 2 above)
Comment (sandrodw3) [reply modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/plugins/rules-sidebar/index.es.js:17]: It's still being used here: https://github.com/liferay/liferay-portal/blob/master/modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/plugins/fields-sidebar/index.es.js.

Thas why I didn't delete it.

File: modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/sidebar/MultiPanelSidebar.es.js (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/sidebar/MultiPanelSidebar.es.js:45]: Super cool idea 😛  done!

**Comment by sandrodw3** [comment]
ci:test:relevant

**Review by p2kmgcl — APPROVED** [review]
OMC (Oh My Cat) THIS IS SO AWESOME

**Comment by sandrodw3** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
Hi @sandrodw3, when add a ddmfield to Create New Fieldset modal, the modal won't work anymore.
The server console will throw 
`2020-12-10 10:31:44.052 ERROR [http-nio-8080-exec-2][status_jsp:807] Query string translates to an empty module paths set`
Actual

**Comment by sandrodw3** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**